lib/iris/experimental/representation.py

```diff
@@ -23,6 +23,7 @@
 from __future__ import (absolute_import, division, print_function)
 from six.moves import (filter, input, map, range, zip)  # noqa
 
+import html
```

Comment:
Given we only need to use `escape` from this library, how about `from html import escape as html_escape` instead.